scripts/us_census/geojsons_low_res/download.py

```diff
@@ -70,7 +70,7 @@ def __init__(self):
         dc.set_api_key('dev')
```

Comment:
Could probably rename it to describe what it's for (getting the **desired** next sub admin area). Feel free to postpone this to another PR.

Reply:
I wonder how this works for weird cases like 5 counties are contained in NYC?